dapps/onboarding.vue/src/components/sign-in/sign-in.vue

```diff
@@ -19,85 +19,54 @@
 
 <template>
   <div class="row h-100" style="display: grid">
-    <evan-onboarding-layout-wrapper 
-      type="sign-in"
-      :step="activeStep">
+    <evan-onboarding-layout-wrapper type="sign-in" :step="activeStep">
       <div class="evan-steps">
-        <h4 class="text-center mt-4 mb-3 text-uppercase font-weight-bold">
-          {{ '_onboarding.sign-in.title' | translate }}
-        </h4>
-
         <div class="step" v-if="activeStep === 0">
-          <p class="text-center mt-3 mb-4" v-html="$t(`_onboarding.sign-in.get-mnemonic-desc`)">
-          </p>
+          <h4
+            class="text-center mt-4 mb-3 text-uppercase font-weight-bold"
+          >{{ '_onboarding.sign-in.title' | translate }}</h4>
```

Comment:
If we are breaking the attributes (i love it ;)) should we also break the inner content?

```suggestion
          >
            {{ '_onboarding.sign-in.title' | translate }}
          </h4>
```

Reply:
The formatting is automatically done via a `prettier` plugin. Ideally we agree on unified settings for it and all apply the same formatting. Should add this to the backlog :)